### PR TITLE
Update translation_domain.rst.inc

### PR DESCRIPTION
**Pull request by enumag**

### CODE REVIEW

File: reference/forms/types/options/translation_domain.rst.inc [hunk 1]
```diff
@@ -1,7 +1,7 @@
 translation_domain
 ~~~~~~~~~~~~~~~~~~
 
-**type**: ``string`` **default**: ``messages``
+**type**: ``string`` or ``false``  **default**: ``messages``
```
Comment (HeahDude) [inline reference/forms/types/options/translation_domain.rst.inc]: Nice catch! Could you please also change the default which is actually `null`.

You should also note that `null` uses the parent form's value, `messages` is the default for the translator, that does not concerns forms.

**Comment by enumag** [comment]
@HeahDude Should I rebase it and change the target branch to 2.7? Or do you have a way to do that on your side like with PRs to Symfony?

**Comment by javiereguiluz** [comment]
Yes, we use the same tool to merge code and docs, so this should be doable by us.

**Comment by xabbuh** [comment]
Thank you @enumag.

**Pull request closed without merging**